### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Arvind644**

### CODE REVIEW

File: components/Team.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```
Comment (adikeshri) [inline components/Team.tsx:1]: Discard changes to this file, already updated

File: pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  	<h1 className="text-6xl font-medium py-8 text-pink-300">oops! Page not found</h1>
+  	 <p className="text-2xl pb-8 px-12 font-medium">Oops! The page you are looking for does not exist. It might have been moved or deleted.</p>
+<button className="bg-gradient-to-r from-pink-600 to-pink-900 hover:from-pink-500 hover:to-orange-500 text-white font-semibold px-6 py-3 rounded-md mr-6">
+	<Link href="/">HOME</Link>
+</button>
+</div>
+</div>
+</div>
+</div>
+  	 </>
```
Comment (adikeshri) [inline pages/404.tsx:20]: Remove commented out code

File: pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    
+    <h1 className="text-6xl font-medium py-8 mt-72 text-pink-300">oops! Page not found</h1>
+     <p className="text-2xl pb-8 px-12 font-medium">Oops! The page you are looking for does not exist. It might have been moved or deleted.</p>
+<button className="bg-gradient-to-r from-pink-600 to-pink-900 hover:from-pink-500 hover:to-orange-500 text-white font-semibold px-6 py-3 rounded-md mr-6">
+  <Link href="/">HOME</Link>
+</button>
+</div>
+
+
+</div>
```
Comment (adikeshri) [inline pages/404.tsx:47]: Fix spaghetti code

**Review by adikeshri — CHANGES_REQUESTED** [review]

File: pages/404.tsx (hunk 3 above)
Comment (Arvind644) [reply pages/404.tsx]: Just edited this page and removed the unwanted code

File: components/Team.tsx (hunk 1 above)
Comment (Arvind644) [reply components/Team.tsx]: Fix that file

File: pages/404.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function Custom404() {
+ return (
+     <>
+      <div className="w-9/12 m-auto py-8 min-h-screen flex items-center justify-center">
+	<Image src="/assets/png/404.jpg" alt="404 page image" width={1500} height={1000}
```
Comment (dntandan) [inline pages/404.tsx:8]: check the image source once. Not loading in deployment preview. Probably /assets/... should be written as assets/...

**Review by dntandan — CHANGES_REQUESTED** [review]

File: pages/404.tsx (hunk 4 above)
Comment (Arvind644) [reply pages/404.tsx:8]: My mistake, I haven't added the image folder yet

Comment (dntandan) [reply pages/404.tsx:8]: LGTM now.

**Review by dntandan — APPROVED** [review]
LGTM

**Pull request merged**